### DISCUSSION TITLE
Issue connecting Google Sheets. Budibase on 2GB droplet

### DISCUSSION THREAD

**Discussion by sladerose** [Budibase/budibase#6823] [Help]
I'm getting this 500 error when trying to connect to my personal Google Sheets account. Does anyone know to fix?

{"message":"Error constructing google authentication strategy: Error: Configuration invalid. Must contain google clientID and clientSecret","status":500}

![image](https://user-images.githubusercontent.com/72910898/179961821-f9b06e72-122d-48a5-9fe7-96b3a59195c6.png)

**Comment by shogunpurple** [comment]
Hi @sladerose - have you added your google cloud platform Client ID and secret TK your organisation settings in the portal?

**Reply by sladerose** [reply]
I have not. Could you please point me in the right direction?

**Reply by danderson2692** [reply]
@sladerose Here's google's documentation on how to create new client IDs: https://developers.google.com/identity/gsi/web/guides/get-google-api-clientid
